assignments/arrays.js

```diff
@@ -63,33 +63,76 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+
+console.log(`Car 33 is a ${inventory[32].car_year}, ${inventory[32].car_make}, ${inventory[32].car_model);
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
+
 let lastCar = 0;
-console.log();
+console.log( {inventory[49].car_make , {inventory[49].car_model} );
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+
+for (i = 0, i < inventory.length, i++)
+{
+
+	carModels[i]= inventory[i].car_model;
+}
+
+console.log(carModels.sort)
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
+
 let carYears = [];
-console.log();
+
+for (i = 0, i < inventory.length, i++)
+
+{
+
+carYears[i] = inventory[i].car_year;
+
+}
+console.log(carYears);
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
-let oldCars =[];
-console.log(); 
+ 
+ let oldCars=[];
+ 
+ for (let i=0; i <inventory.length; i++){
+ 	 
+ 	 if(inventory[i].car_year < 2000){ 
+
+ 		
+ 	 
+     oldCars[i]=(inventory[i].car_year);
```

Comment:
```suggestion
     oldCars.push((inventory[i].car_year));
```
You need to use the `push()` method here because the array starts out empty and does not have a value at the given indexes already